src/amo/pages/Guides/index.js

```diff
@@ -0,0 +1,193 @@
+  },
+|};
+
+type InternalProps = {|
+  ...Props,
+  clientApp: string,
+  dispatch: DispatchFunc,
+  errorHandler: ErrorHandlerType,
+  slug: string,
+  guids: Array<string>,
+  i18n: I18nType,
+  lang: string,
+|};
+
+type SectionsType = {|
+  addonGuid: string,
+  header: string,
+  description: string,
+  addonCustomText: string,
+  exploreMore: string,
+  exploreUrl: string,
+|};
+
+type GuideType = {|
+  title: string,
+  introText: string,
+  icon: string,
+  sections: Array<SectionsType>,
+|};
+
+export const getContent = (slug: string, i18n: I18nType): GuideType | null => {
+  switch (slug) {
+    case 'privacy':
+      return {
+        title: i18n.gettext('Stay Safe Online'),
+        introText: i18n.gettext(
+          `The web is a wonderful but wild place. Your personal data can be used
+           without your consent, your activities spied upon, and your passwords
+           stolen. Fortunately, extensions can help fortify your online privacy
+           and security.`,
+        ),
+        icon: 'stop-hand',
+        sections: [
+          // Bitwarden free password manager
+          {
+            addonGuid: '{446900e4-71c2-419f-a6a7-df9c091e268b}',
+            header: i18n.gettext('Create and manage strong passwords'),
+            description: i18n.gettext(
+              `Password managers can help you create secure passwords, store your
+               passwords (safely) in one place, and give you easy access to your
+               login credentials wherever you are.`,
+            ),
+            addonCustomText: i18n.gettext(
+              `Fully encrypted password protection. Store your data securely and
+               access logins across devices.`,
+            ),
+            exploreMore: i18n.gettext(
+              'Explore more %(linkStart)spassword manager%(linkEnd)s staff picks.',
+            ),
+            exploreUrl: '/collections/mozilla/password-managers/',
+          },
+        ],
+      };
+    default:
+      return null;
+  }
+};
+
+export class GuidesBase extends React.Component<InternalProps> {
+  constructor(props: InternalProps) {
+    super(props);
+
+    const { errorHandler, i18n, match } = this.props;
+    const { slug } = match.params;
+    const content = getContent(slug, i18n);
+
+    if (!content) {
+      return;
+    }
+
+    const guids = content.sections.map((section) => section.addonGuid);
+
+    this.props.dispatch(
+      fetchGuidesAddons({
+        guids,
+        errorHandlerId: errorHandler.id,
+      }),
+    );
+  }
+
+  getGuidesSections = (
+    sections: Array<SectionsType>,
+  ): React.ChildrenArray<React.Node> => {
+    const { clientApp, i18n, lang } = this.props;
+
+    return sections.map((section) => {
+      const exploreMoreLink = i18n.sprintf(section.exploreMore, {
+        linkStart: `<a class="Guides-section-explore-more-link" href="/${lang}/${clientApp}${
+          section.exploreUrl
+        }">`,
+        linkEnd: '</a>',
+      });
```

Comment:
Do you have an issue opened for that? (I think I saw something a while ago) Make sure there is an issue, no need to reference it here though.

Reply:
yep it WAS there but it's missing now:/ I will add this back, thanks!

Reply:
yeh there IS an issue so I think I'll reference that here:)